spi_nand_flash/diskio/diskio_nand.c

```diff
@@ -106,6 +132,11 @@ DRESULT ff_nand_ioctl(BYTE pdrv, BYTE cmd, void *buff)
         ESP_LOGV(TAG, "sector size=%d", *((WORD *)buff));
         break;
     }
+#if FF_USE_TRIM
+    case CTRL_TRIM:
+        return ff_nand_trim(pdrv, *((DWORD *)buff), //start_sector
+                            (*((DWORD *)buff + 1) - * ((DWORD *)buff) + 1)); //sector_count
```

Comment:
Just a suggestion for better readability:
```suggestion
    {
        DWORD start_sector = *((DWORD *)buff);
        DWORD end_sector = *((DWORD *)buff + 1) + 1;
        DWORD sector_count = end_sector - start_sector;
        return ff_nand_trim(pdrv, start_sector, sector_count);
    }                            
```
and